standalone/detectors/fasttrack/include/MemoryPool.h

```diff
@@ -0,0 +1,61 @@
+#ifndef MEMORY_POOL_HEADER_H
+#define MEMORY_POOL_HEADER_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define MEMORY_POOL_HEADER_H
```